index.html

```diff
@@ -19,7 +19,7 @@
 		<meta property="og:url" content="https://mytimetable.csclub.org.au/" />
 		<meta
 			property="og:description"
-			content="A drag-and-drop timetable planner for University of Adelaide students."
+			content="MyTimetable is a simple drag-and-drop timetable planner for University of Adelaide students. Easily organise your course classes and create the perfect timetable with this tool by the CS Club Open Source Team."
```

Comment:
The longer description was just needed for the top meta description, the og and twitter descriptions have lower character limits, so make them just use the first sentence.